test/WebJobs.Script.Tests.Integration/TestScripts/Proxy/proxies.json

```diff
@@ -0,0 +1,25 @@
+{
+  "proxies": {
+    "test": {
+      "matchCondition": {
+        "route": "/myproxy",
+        "methods": [
+          "GET"
+        ]
+      },
+      "backendUri": "http://localhost/api/httptrigger"
```

Comment:
Yes. Note that this currently will require two hops. i.e. the request first goes to proxies, then to functions (via frontend). The optimization that will come later is that we get a handle for a FunctionExecutor that we can call from functions to execute this functions. This is what we will use to have function extensibility also. These changes are not part of this CR, but will follow post GA.

---
In reply to: [132552230](https://github.com/Azure/azure-webjobs-sdk-script/pull/1749#discussion_r132552230) [](ancestors = 132552230)